documentation/assemblies/configuring/assembly-logging-configuration.adoc

```diff
@@ -0,0 +1,68 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='external-logging_{context}']
+= Logging configuration
+
+[role="_abstract"]
+Configure logging levels in the custom resources of Kafka components and Strimzi Operators.
+You can specify the logging levels directly in the `spec.logging` property of the resource YAML.
+Or you can define the logging properties in a ConfigMap that's referenced in the YAML using the `configMapKeyRef` property.
+
+The advantages of using a ConfigMap are that the logging properties are maintained in one place and are accessible to more than one resource.
```

Comment:
You can also re-use the configmap for multiple resources.

Reply:
Added.